enhancements/deprecated-bundles.md

```diff
@@ -95,32 +84,56 @@ we then deprecate version 1.3.0 using the following command
 Now, in `quay.io/my/index:v4.7` the above channel looks like:
 
 ```
-    1.4.0 -- replaces -> 1.3.0 [deprecated]
+    1.4.0 -- replaces -> 1.3.0 [deprecated] -- replaces -> 1.2.0 [deprecated] -- replaces -> 1.1.0 [deprecated] 
 ```
 
-Deprecated versions are upgradable but not installable. To achieve this, we will modify the operator-registry apis used by OLM to determine what can be installed.
+Deprecated versions are upgradable but not installable.
 
 #### Story 2
 
 As an index maintainer I want to be able to undo the above operation.
 
 ##### Implementation Details
 
-Index maintainers should revert to the index image used in the original command. If the tag is overwritten, they must keep track of the shas of the index images they modify in order to guarantee recoverability. If the image is, for some reason, not available, the index maintainer can remove the entire package using `opm index rm` and re-add all the bundles for that package.
+A flag called `--undo` is added to the deprecation command that removes the deprecated property on the bundles specified.
```

Comment:
in general I agree and I will add a point about this but wrt the pipelines, since they keep a list of all deprecated bundles, presumably when you "undeprecate" you also only remove that one bundle from the deprecation list repo which would not have the above side effect since the other bundle would still be listed and its deprecation would be affected right?

Reply:
> presumably when you "undeprecate" you also only remove that one bundle from the deprecation list repo which would not have the above side effect since the other bundle would still be listed and its deprecation would be affected right?

if they apply everything in the right order, yes.  but not if they apply "undeprecations" last.